sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/BatchSpannerRead.java

```diff
@@ -18,12 +18,12 @@
 package org.apache.beam.sdk.io.gcp.spanner;
 
 import com.google.auto.value.AutoValue;
-import com.google.cloud.spanner.BatchReadOnlyTransaction;
-import com.google.cloud.spanner.Partition;
-import com.google.cloud.spanner.ResultSet;
-import com.google.cloud.spanner.Struct;
-import com.google.cloud.spanner.TimestampBound;
+import com.google.cloud.spanner.*;
```

Comment:
Could you run ./gradlew spotlessApply ?

I think the preferred style is to avoid wildcard imports

Reply:
Yes, done!